cmd/jujud/agent/machine.go

```diff
@@ -1208,6 +1208,7 @@ func (a *MachineAgent) startModelWorkers(controllerUUID, modelUUID string) (work
 		CharmRevisionUpdateInterval: 24 * time.Hour,
 		InstPollerAggregationDelay:  3 * time.Second,
 		StatusHistoryPrunerInterval: 5 * time.Minute,
+		ActionPrunerInterval:        2 * 7 * 24 * time.Hour, // once every a fortnight
```

Comment:
It is needed since some collections marshal to Unix Nanoseconds while other use the default go `time.Time` marshaling. Both which yield  different results.